<?xml version="1.0" encoding="utf-8"?>

<launch>

  <node name="object_detection_py" output="screen" pkg="object_detection" type="object_detection.py">
    <param name="image_width" type="int" value="640" />
    <param name="image_height" type="int" value="480" />
    <param name="image_topic" value="/usb_cam1/image_raw/compressed" />
  </node>

</launch>